DFC.ServiceTaxonomy.ApiFunction/Function/Execute.cs

```diff
@@ -62,6 +69,9 @@ public Execute(IOptionsMonitor<ServiceTaxonomyApiSettings> serviceTaxonomyApiSet
                 var cypherPathParameters = GetCypherPathParameters(cypherModel, req.Path, log);
                 var cypherQueryParameters = GetCypherQueryParameters(cypherModel, req.Query, await requestBodyTask, log, cypherPathParameters);
 
+                //Add the host in to all cypher queries
+                cypherQueryParameters.Add("host", host);
```

Comment:
I was thinking we'd add a param to the Cypher query config json, but I think it's OK to add it as a (sometimes unused) param to all queries.
We can always refactor it later if it makes sense, e.g. if we add more calculated params, we might only want to pay the cost of creating the params, if they're actually used by the current query.